### PR TITLE
Timeseries updates

### PR DESCRIPTION
**Pull request by jotaylor**
This PR includes two updates:

1. Updates required for serendipitous timeseries. This only changes the format of the timeseries YAML files to add more details. 
2. 2nd epoch information for monitoring stars

### CODE REVIEW

**Comment by jotaylor** [comment]
I merged the `dr6_tts` branch into this one, as it was needed for complete testing. So, maybe this should be reviewed after that branch is merged (I think there are still some pending updates to that branch)

File: utils/data/timeseries/tss_template_nocomments.yaml
Comment (Rplesha) [inline utils/data/timeseries/tss_template_nocomments.yaml:1]: The name of this file makes me wonder if it should be included vs. tts_template

**Review by Rplesha — APPROVED** [review]
So many yaml files. Approve if the "nocomments" file should be included. If not, I'm happy to review again and approve

File: utils/data/timeseries/tss_template_nocomments.yaml
Comment (jotaylor) [reply utils/data/timeseries/tss_template_nocomments.yaml:1]: my rationale was maybe that the tss_template.yaml is really long, and this one can be used for people who don't want to wade through lots of comments? But now that I think twice about it, I'm wondering if that is overkill 😂

Comment (Rplesha) [reply utils/data/timeseries/tss_template_nocomments.yaml:1]: It's long, but I don't think it's *that* long, personally. I never use or update these files though, so I defer to those who do!

Comment (jotaylor) [reply utils/data/timeseries/tss_template_nocomments.yaml]: @Rplesha Ok, I removed the extra template file :)

**Review by Rplesha — APPROVED** [review]

**Pull request merged**
